src/process_utility.c

```diff
@@ -1697,9 +1697,58 @@ process_altertable_end_subcmd(Hypertable *ht, Node *parsetree, ObjectAddress *ob
 					 errmsg("hypertables do not support inheritance")));
 		case AT_SetStatistics:
 		case AT_SetLogged:
+		case AT_SetStorage:
+		case AT_ColumnDefault:
+		case AT_SetNotNull:
+		case AT_DropNotNull:
+		case AT_AddOf:
+		case AT_DropOf:
+#if PG10
+		case AT_AddIdentity:
+		case AT_SetIdentity:
+		case AT_DropIdentity:
+#endif
 			/* handled by default recursion */
 			break;
+		case AT_EnableRowSecurity:
+		case AT_DisableRowSecurity:
+		case AT_ForceRowSecurity:
+		case AT_NoForceRowSecurity:
+			/* should not recurse to chunks */
```

Comment:
Do these comments apply to all cases above or just the last? This is not clear. Maybe you can be more explicit and say it applies to all RowSecurity commands? Same issue with other comments here.